.github/workflows/build-binaries.yml

```diff
@@ -50,6 +51,14 @@ jobs:
           ./node_modules/@oclif/dev-cli/bin/run pack -t="${{ matrix.package-targets }}"
           mv ./dist/version ./dist/${{ matrix.os }}-version
           mv ./dist/coveo-${{ env.tag }}/coveo-${{ env.tag }}.tar.gz ./dist/coveo-${{ env.tag }}/${{ matrix.os }}-coveo-${{ env.tag }}.tar.gz
+      - name: Sign Executable
```

Comment:
How do you see us mitigating the certificate expiration ?

Versus having to remember it every year ? :)

Reply:
I don't really have a fool-proof solution on that end.
I know the certificate will expire in June 2024 and that it's used by Maestro and our Nugget packages too:
https://coveord.atlassian.net/wiki/spaces/CI/pages/2514747576/Maestro+-+Code+Signing+Certificate+PFX